src/main/java/com/ldtteam/blockout/hooks/HookScreen.java

```diff
@@ -0,0 +1,88 @@
+package com.ldtteam.blockout.hooks;
+
+import com.ldtteam.blockout.BOScreen;
+import com.ldtteam.blockout.hooks.TriggerMechanism.Type;
+import com.ldtteam.blockout.views.ScrollingList;
+import com.mojang.blaze3d.matrix.MatrixStack;
+
+/**
+ * Screen wrapper.
+ */
+public class HookScreen extends BOScreen
```

Comment:
this class is missing @Overrides

Reply:
nah, those methods are new methods, if you mean some concrete methods then name them (I think I have covered all of them)

Reply:
ah okay, those are missing some javadoc then =D, I thought the render method at least

Reply:
kk